site/docs/status.md

```diff
@@ -0,0 +1,362 @@
+| Update table properties     | Y    | Y         | Y    | N  |
+| Replace sort order          | Y    | N         | N    | N  |
+| Update table location       | Y    | N         | N    | N  |
+| Append data files           | Y    | N         | N    | N  |
+| Rewrite files               | Y    | N         | N    | N  |
+| Rewrite manifests           | Y    | N         | N    | N  |
+| Overwrite files             | Y    | N         | N    | N  |
+| Row delta                   | Y    | N         | N    | N  |
+| Delete files                | Y    | N         | N    | N  |
+| Update statistics           | Y    | N         | N    | N  |
+| Update partition statistics | Y    | N         | N    | N  |
+| Expire snapshots            | Y    | N         | N    | N  |
+| Manage snapshots            | Y    | N         | N    | N  |
+
+### Table Spec V2
+
+| Operation                   | Java | PyIceberg | Rust | Go |
+|-----------------------------|------|-----------|------|----|
+| Update schema               | Y    | N         | N    | N  |
+| Update partition spec       | Y    | N         | N    | N  |
+| Update table properties     | Y    | Y         | Y    | N  |
+| Replace sort order          | Y    | N         | N    | N  |
+| Update table location       | Y    | N         | N    | N  |
+| Append data files           | Y    | N         | N    | N  |
+| Rewrite files               | Y    | N         | N    | N  |
+| Rewrite manifests           | Y    | N         | N    | N  |
+| Overwrite files             | Y    | N         | N    | N  |
+| Row delta                   | Y    | N         | N    | N  |
+| Delete files                | Y    | N         | N    | N  |
+| Update statistics           | Y    | N         | N    | N  |
+| Update partition statistics | Y    | N         | N    | N  |
+| Expire snapshots            | Y    | N         | N    | N  |
+| Manage snapshots            | Y    | N         | N    | N  |
+
+## Table Read Operations
+
+### Table Spec V1
+
+| Operation           | Java | PyIceberg | Rust | Go |
+|---------------------|------|-----------|------|----|
+| Plan with data file | Y    | N         | Y    | N  |
+| Read data file      | Y    | N         | Y    | N  |
+
+### Table Spec V2
+
+| Operation                  | Java | PyIceberg | Rust | Go |
+|----------------------------|------|-----------|------|----|
+| Plan with data file        | Y    | N         | Y    | N  |
+| Plan with position deletes | Y    | N         | N    | N  |
+| Plan with equality deletes | Y    | N         | N    | N  |
+| Read data file             | Y    | N         | Y    | N  |
+| Read with position deletes | Y    | N         | N    | N  |
+| Read with equality deletes | Y    | N         | N    | N  |
+
+## Table Write Operations
+
+### Table Spec V1
+
+| Operation   | Java | PyIceberg | Rust | Go |
+|-------------|------|-----------|------|----|
+| Append data | Y    | N         | Y    | N  |
```

Comment:
@Fokko Pyiceberg supports writes, correct?

Reply:
Yes, for both V1 and V2 :)

Reply:
```suggestion
| Append data | Y    | Y         | Y    | N  |
```